packages/ui/src/Table.tsx

```diff
@@ -60,7 +60,7 @@ export interface TableProps extends React.ComponentPropsWithoutRef<'table'> {
   /**
    * Required data prop, should be array of objects.
    */
-  data: TableData[]
+  data: any
```

Comment:
@aleksaToljic  I set this temporary until I find a solution that works.
Here I got an error when displaying fetched data because it doesn't accept type WalletBalance. Error: (Type 'WalletBalance[]' is not assignable to type 'TableData[]')